### PR TITLE
displaying error message.

### PR DESCRIPTION
**Pull request by laxmankhanal**

### CODE REVIEW

File: Vaccination/View/CustomVaccineFormCell.swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+class CustomVaccineFormCell: UITableViewCell {
+
+    @IBOutlet var textField: UITextField!
+    @IBOutlet var errorMessageLabel: UILabel!
+}
```
Comment (sanjaymhj) [inline Vaccination/View/CustomVaccineFormCell.swift]: line at the EOF and  before the ending bracket

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 2]
```diff
@@ -2,12 +2,14 @@ import UIKit
 
 class AddChildFormTVC: UITableViewController, UITextFieldDelegate {
 
+    @IBOutlet var doneButton: UIButton!
+
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: why extra line here?

File: Vaccination/View/CustomVaccineFormCell.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/View/CustomVaccineFormCell.swift]: :ok:

File: Vaccination/ViewController/AddChildFormTVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/AddChildFormTVC.swift]: :ok:

File: Vaccination/View/ChildFormViewCell.swift [hunk 3]
```diff
@@ -3,5 +3,6 @@ import UIKit
 class ChildFormViewCell: UITableViewCell {
 
     @IBOutlet var textField: UITextField!
+    @IBOutlet var errorMessageLabel: UILabel!
 
 }
```
Comment (sanjaymhj) [inline Vaccination/View/ChildFormViewCell.swift]: need line at EOF

File: Vaccination/View/CustomVaccineFormCell.swift [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+class CustomVaccineFormCell: UITableViewCell {
+
+    @IBOutlet var textField: UITextField!
+    @IBOutlet var errorMessageLabel: UILabel!
+    
+}
```
Comment (sanjaymhj) [inline Vaccination/View/CustomVaccineFormCell.swift]: also need line at EOF

File: Vaccination/ViewController/MainVC.swift [hunk 5]
```diff
@@ -247,7 +287,7 @@ extension MainVC: UITableViewDataSource {
                 } else {
                     return tableData.count
                 }
-        
+
             default:
                 return 1
         }
```
Comment (sanjaymhj) [inline Vaccination/ViewController/MainVC.swift]: swiftlint autocorrect it dude

File: Vaccination/View/ChildFormViewCell.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/View/ChildFormViewCell.swift]: :0k:

File: Vaccination/View/CustomVaccineFormCell.swift (hunk 4 above)
Comment (laxmankhanal) [inline Vaccination/View/CustomVaccineFormCell.swift]: :ok:

File: Vaccination/ViewController/MainVC.swift (hunk 5 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/MainVC.swift]: 🆗 :

**Comment by sanjaymhj** [comment]
:+1:

**Pull request merged**